test/parallel/test-common.js

```diff
@@ -108,3 +108,23 @@ const HIJACK_TEST_ARRAY = [ 'foo\n', 'bar\n', 'baz\n' ];
   common[`restoreStd${txt}`]();
   assert.strictEqual(originalWrite, stream.write);
 });
+
+// hijackStderr and hijackStdout again
+// for console
+[ 'err', 'out' ].forEach((txt) => {
```

Comment:
Optional: do `[['err', 'error'], ['out', 'log']].forEach(([ext, method]) => {`
instead of `console[txt === 'err' ? 'error' : 'log']('test');`